### ISSUE TITLE
Create an API endpoint to save data on PostgreSQL

### ISSUE THREAD

**Issue by ruphy** [italia/daf-dataportal#43]
Create an API endpoint, linked to the ingestion form, which will save the data on an external PostgreSQL database.

**Comment by simonacca** [comment]
Would [postgrest](https://postgrest.com/en/v4.3/) suffice?

**Comment by fabiofumarola** [comment]
PostgREST is a standalone web server that turns your PostgreSQL database directly into a RESTful API. The structural constraints and permissions in the database determine the API endpoints and operations.

**Comment by giux78** [comment]
yes for me is perfect. The goal is to have some endpoints where data csv and json go into postgres and can be queried easily. Seems perfect. Maybe can be added more functionality on top of the API.

**Comment by fabiofumarola** [comment]
ok, cool

**Comment by ermannob** [comment]
What about using PostgreSQL Json query engine over No-sql data? The form can be serialized to JSON and saved into a PostgreSQL's json field. I think is viable and more flexible compared to structuring the data. What kind of use cases do you have in mind?

**Comment by giux78** [comment]
Yes, @ermannob the use case is to use postgres instead of the big data layer for working locally with DAF without connecting to a cluster. We have many tools (metabase, superset, jupyter via spark), api ecc that can interact with postgres and could be useful to have a single entry point for the data
